crates/turborepo-filewatch/src/globwatcher.rs

```diff
@@ -30,7 +30,7 @@ pub struct GlobSet {
 impl GlobSet {
     pub fn as_inputs(&self) -> Vec<String> {
         let mut inputs: Vec<String> = self.include.keys().cloned().collect();
-        inputs.extend(self.exclude_raw.iter().cloned());
+        inputs.extend(self.exclude_raw.iter().map(|s| format!("!{}", s)));
```

Comment:
Should we add a comment noting that the exclude globs do not contain `!` in the `GlobSet` type? That might be good to document

Reply:
Done.